### DISCUSSION TITLE
Does using renovate require a project to be open source?

### DISCUSSION THREAD

**Discussion by maxs-rose** [renovatebot/renovate#20994] [Request Help] [answered]
We are looking use renovate at my company but our projects are not allowed to be open source.

As renovate is under the AGPL license would using renovate on a project the require that project to become open source?

**Comment by rarkins** [comment, marked as answer]
No it does not

**Reply by maxs-rose** [reply]
When using renovate what is the licensing?

Since I am sure someone will come asking eventually.

**Reply by rarkins** [reply]
Renovate is AGPL licensed, as you already know. Your code is whatever license you decide - using Renovate does not change it.
